react/assets/icons/photos.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 24 24" aria-labelledby="imageIconTitle" stroke="#2329D6" stroke-width="1" stroke-linecap="square" stroke-linejoin="miter" fill="none" color="#2329D6"> <title id="imageIconTitle">Image</title> <rect width="18" height="18" x="3" y="3"/> <path stroke-linecap="round" d="M3 14l4-4 11 11"/> <circle cx="13.5" cy="7.5" r="2.5"/> <path stroke-linecap="round" d="M13.5 16.5L21 9"/> </svg>
```

Comment:
Where did this SVG come from and where is it being used?

Reply:
Coming from Ikonate

Used here in this PR: 
<img width="441"  src="https://user-images.githubusercontent.com/8842023/116434335-3d055680-a842-11eb-93c6-3eb95bba9e6b.png">

Tried to explain with [my comment](https://github.com/ripe-tech/ripe-components-react-native/pull/168#issuecomment-826899863) above

Reply:
I thought we already had everything from Ikonate, weird